basic/helloworld-groovy/src/main/groovy/org/springframework/integration/samples/helloworld/PollerConfig.groovy

```diff
@@ -42,20 +40,8 @@ class PollerConfig {
 				{ e -> e.poller(Pollers.fixedDelay(20000).maxMessagesPerPoll(2)) })
 				{
 					log LoggingHandler.Level.INFO, 'org.springframework.integration.samples.helloworld'
-					channel { queue 'loggerChannel' }
+					channel(new NullChannel())
```

Comment:
Oh! No `nullChannel()` in Groovy DSL.
Any chances that you can contribute one into `GroovyIntegrationFlowDefinition` ?